backend/.mockery.private.yml

```
@@ -415,6 +415,22 @@ packages:
       pkgname: attestation
       filename: "{{.InterfaceName}}_mock_test.go"
+      
+  github.com/thunder-id/thunderid/internal/scim:
+    config:
+      all: true
+      dir: internal/scim
+      structname: '{{.InterfaceName}}Mock'
+      pkgname: scim
+      filename: "{{.InterfaceName}}_mock_test.go"
+
+  github.com/thunder-id/thunderid/internal/scim:
```

YAMLlint (1.37.1)

[error] duplication of key "github.com/thunder-id/thunderid/internal/scim" in mapping

(key-duplicates)